addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -592,11 +599,14 @@
 
         <t t-foreach="category_data" t-as="category">
             <div class="mb-2" t-if="(category['slides'] or channel.can_publish) and (search_category and search_category.id == category['id'] or not search_category)">
-                <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['name'] != 'Uncategorized'">
-                    <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
+                <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
+                <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['id'] > 0 and query_string != '?search_uncategorized=1'">
```

Comment:
`t-if="category['id'] and query_string != '?search_uncategorized=1'`  should do the trick?

Testing > 0 when it actually contains "False" is strange.